src/containers/ViewerPanel/index.tsx

```diff
@@ -179,11 +179,11 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
     public componentDidUpdate(prevProps: ViewerPanelProps) {
         const { status, error } = this.props;
         const current = this.centerContent.current;
-        if (
+        const isNewError: boolean =
             status === VIEWER_ERROR &&
-            prevProps.status !== VIEWER_ERROR &&
-            error.message
-        ) {
+            error.message !== prevProps.error.message;
```

Comment:
This change is necessary to display multiple errors from the same file load (e.g. when a trajectory is missing 3 geometry files, there are 3 error message passed up from the viewer but only the first one was displaying before)